core-java-modules/core-java-20/src/test/java/com/baeldung/convertpaths/RelativePathConverterUnitTest.java

```diff
@@ -0,0 +1,51 @@
+package com.baeldung.convertpaths;
+
+import org.junit.Test;
+
+public class RelativePathConverterUnitTest {
+
+    @Test
+    public void givenRelativePath_whenConvertingToAbsolutePath_thenPrintOutput() {
```

Comment:
thanks for adjusting the tests to their conventions. 
However, tests are not really testing anything, right? 

What about adding assertions to the expected values they should be? :)